src/start_server.ts

```diff
@@ -113,6 +116,22 @@ export function getApp(options: ServerOptions) {
   return app;
 }
 
+/**
+ * Open the given web page URL. If no browser keyword is provided, `opn` will use
+ * the user's default browser.
+ */
+function openWebPage(url: string, withBrowser?: string) {
+  let openOptions = {
+    app: withBrowser
+  };
+  opn(url, openOptions, (err) => {
+    if (err) {
+      console.log(`Error launching "${openOptions.app || 'default web browser'}".`);
```

Comment:
I'm going to change it to log the error and continue, that seems to serve more people/use cases